packages/trpc/server/routers/viewer/eventTypes.ts

```diff
@@ -406,7 +408,29 @@ export const eventTypesRouter = router({
   create: authedProcedure.input(createEventTypeInput).mutation(async ({ ctx, input }) => {
     const { schedulingType, teamId, ...rest } = input;
     const userId = ctx.user.id;
+    // Get Users default conferncing app
+    const usersMetadata = ctx.user.metadata;
+
+    const defaultConferencingData = userMetadataSchema.parse(usersMetadata)?.defaultConferencingApp;
```

Comment:
Remove redundant constant in favour of clarity.

```suggestion
    const defaultConferencingData = userMetadataSchema.parse(ctx.user.metadata)?.defaultConferencingApp;
```